server/protocols/jmap-draft-integration-testing/rabbitmq-jmap-draft-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQSwiftFilterTest.java

```diff
@@ -35,7 +35,7 @@
     public DockerCassandraRule cassandra = new DockerCassandraRule();
 
     @Rule
-    public CassandraRabbitMQSwiftJmapTestRule rule = CassandraRabbitMQSwiftJmapTestRule.defaultTestRule();
+    public CassandraRabbitMQAwsS3JmapTestRule rule = CassandraRabbitMQAwsS3JmapTestRule.defaultTestRule();
```

Comment:
I missed it.

But it's plenty stupid: Why play tests that don't require blobStorage on 2 blobstore provider, except to be loosing valuable CI time? I'm dropping this RabbitMQSwiftFilterTest too :-)

Reply:
that's what we call integration test... How to know if something is working if not tested?

Reply:
Well, we can test dozens of other things, but we would be taking ages doing so. The question is "does it bringsvalyes?". We already have tests demonstrating "basic features works on top of swift". Filters don't exercise blobStore. Hence this test class is not bringing any value yet brings confusion (proof: i did not get the intent at first), and increase tesing time.

Exactly the kind of things I want to remove.

Reply:
Disclaimer: I'm not saying we must keep this test, I'm just trying to explain why it exists, and so maybe why it could be kept.

How do you check that the "filter" feature is correctly bound in this product?

Reply:
We check that jmap is correctly wired. Filters are part of jmap. There are dozens of other features whose wiring is not tested on top of swift.